OneCommunityGlobal/HighestGoodNetworkApp
# Luis fix weekly summary bug

# Description
Weekly summary allowed summaries less than 50 words from being submitted. 
<img width="797" alt="Screenshot 2023-09-21 at 7 38 14 PM" src="https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/22454953/739143a3-4d47-4928-a559-4d19f1e86f38">

video of bug:
https://drive.google.com/file/d/1EQltPvUFPeB7nuap_VmXoinHfeHJMdDK/view



## Related PRS (if any):

…

## Main changes explained:
Got the word count from the Editor since it displayed the word count. 
I added another formElement parameter called wordCount and added it to the schema for JOI to test
Since the current Regex checks for whitespaces I added another check for word count, since this is why summaries less than 50 words were being submitted. The regex was only counting the white spaces.



## How to test:
1. check into current branch
2. do `npm install` and `...` to run this PR locally
3. log as admin user and volunteer. 
4. go to dashboard→ summary
5. Fill out the weekly summary with less than 50 words then attempt to submit it. 
6. Complete the summary by adding 50 words and should let you submit it. 
*Note make sure the user has a dropbox link otherwise you'll need add one when submitting. 


## Screenshots or videos of changes:


https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/22454953/ee4b6f56-bf0c-4d3a-8d32-bb005f75afb7


https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/22454953/6f167f9e-478b-4d0c-9620-464a8f42d594






## Note:
the following errors appear but are unrelated to the PR. when submitting via a volunteer account.
<img width="403" alt="Screenshot 2023-09-21 at 7 46 12 PM" src="https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/22454953/978f612f-bc59-4815-b4d9-242666e99304">



the following appeared for submitting via an admin
<img width="428" alt="Screenshot 2023-09-21 at 7 48 51 PM" src="https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/22454953/55120a4a-907a-4b17-84a8-74899824fd69">
